apps/libs/digest.h

```diff
@@ -0,0 +1,34 @@
+#ifndef TOCK_DIGEST_H
+#define TOCK_DIGEST_H
+
+#include <stdlib.h>
+
+// TODO These need to be standardized
+#define HOTEL_DRIVER_DIGEST 2
```

Comment:
I feel like we should be standardizing these across chips, because otherwise there's going to just be no way of writing crossplatform programs/examples.